pkg/cmd/server/start/master_args.go

```diff
@@ -254,6 +254,7 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 		if builtInKubernetes {
 			config.KubeletClientInfo.CA = admin.DefaultRootCAFile(args.ConfigDir.Value())
 			config.KubeletClientInfo.ClientCert = kubeletClientInfo.CertLocation
+			config.ServiceAccountConfig.MasterCA = admin.DefaultRootCAFile(args.ConfigDir.Value())
```

Comment:
Since it's tightly coupled to the master using TLS, I'd rather keep it in the top stanza.  It is getting set because we're using TLS and we're starting kube and there's an existing stanza just for that case.